### ISSUE TITLE
Show stack trace for inline evaluations that go wrong

### ISSUE THREAD

**Issue by PEZ** [BetterThanTomorrow/calva#434]
As an infrequent user of the REPL window, I do have one use case where I always use it: When an evaluation fails and I don't understand the error message. Then I evaluate the same thing to the REPL window to get the stack trace it prints. It would be nice to have this help automatically, so that when an editor inline evaluation fails, the stack trace is always ready to be shown in the REPL window. So something like this:

1. An evaluation fails
1. The error is printed in the REPL window
1. The window is revealed without the editor losing focus. (Automatically revealing the window might be something that should be configurable) Or not happening when the window Is in the same group. We need to also figure out what to do if the window is not opened.
1. In the REPL window I can click to show the stack trace.

Related: #424

**Comment by stefan-toubia** [comment]
This should be pretty easy to implement after #424, I will keep this in mind.

**Comment by zcaudate** [comment]
I'd like to have something of this type as well. The errors when in-line evaling are not enough and I'm having to constantly fall back to emacs to figure out what is wrong.
